packages/core/tests/action-menu.test.ts

```diff
@@ -0,0 +1,119 @@
+import type { SubjectMessage } from '../../../tests/transport/SubjectInboundTransport'
```

Comment:
Can we move this to the action menu module directory tests and call it `action-menu.e2e.test.ts`? I'd like to start co-locating tests (even e2e) with their module are more contained.

Reply:
Good idea, that will be nice to easily to separate modules from core.